app/uk/gov/hmrc/exports/repositories/SubmissionRepository.scala

```diff
@@ -43,7 +43,8 @@ class SubmissionRepository @Inject() (val mongoComponent: MongoComponent)(implic
       mongoComponent = mongoComponent,
       collectionName = "submissions",
       domainFormat = Submission.format,
-      indexes = SubmissionRepository.indexes
+      indexes = SubmissionRepository.indexes,
+      replaceIndexes = true
```

Comment:
@devs Please note we will need a 2nd deployment to TDR and production where this flag has to be removed.

Reply:
I have an idea for this that I'd like to run past you Lucio (to see what you think of it)